boards/arm/96b_neonkey/96b_neonkey.dts

```diff
@@ -35,5 +35,6 @@
 };
 
 &i2c3 {
-	clock-frequency = <I2C_BITRATE_FAST>;
+	status = "ok";
```

Comment:
Is this LED driver for a chip that is built-in to the board?

If so, this seems fine, but otherwise, why not move this into a DT overlay in your sample, which  gets added to DTC_OVERLAY_FILE by the CMakeLists.txt in the sample?

Reply:
Yes, this chip is present on the Neonkey board.